cmd/descheduler/app/options/options.go

```diff
@@ -19,20 +19,29 @@ package options
 
 import (
 	"github.com/spf13/pflag"
+
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 	clientset "k8s.io/client-go/kubernetes"
-
 	"k8s.io/component-base/logs"
+
 	"sigs.k8s.io/descheduler/pkg/apis/componentconfig"
 	"sigs.k8s.io/descheduler/pkg/apis/componentconfig/v1alpha1"
 	deschedulerscheme "sigs.k8s.io/descheduler/pkg/descheduler/scheme"
 )
 
+const (
+	DefaultDeschedulerPort = 10258
```

Comment:
Can you mention in somewhere that port `10258` is used for metrics by default and users can use command line flag `--secure-port` to custom port number? It's not easy to get this info from the help command.

Reply:
Added some comments in `--disable-metrics` flag about it.